packages/workspace/src/generators/preset/preset.ts

```diff
@@ -252,7 +252,7 @@ async function createPreset(tree: Tree, options: Schema) {
       directory: join('apps', options.name),
```

Comment:
There are some console logs up here... can you remove those please?

Reply:
removed